layouts/partials/sidebar.html

```diff
@@ -1,11 +1,12 @@
 <aside class="sidebar" itemscope="itemscope" itemtype="http://schema.org/WPSideBar">
 	{{ partial "widgets/search.html" . }}
 	{{ partial "widgets/recent.html" . }}
+	{{ partial "widgets/recent-events.html" . }}
 	{{ partial "widgets/categories.html" . }}
 	{{ partial "widgets/social.html" . }}
 	{{ partial "widgets/taglist.html" . }}
 
 	{{- if and (not .Site.Params.widgets.search) (not .Site.Params.widgets.recent_articles) (not .Site.Params.widgets.categories) (not .Site.Params.widgets.social) (not .Site.Params.widgets.taglist) }}
 	<p class="sidebar__warning"><strong>WARNING:</strong><br>Please activate at least one sidebar widget.</p>
 	{{- end }}
-</aside>
+</aside>
```

Comment:
Trailing newline please (is your editor configured to remove them? If so, please change that -- lines aren't complete without their terminating newline character)

Reply:
This seems to be a recurring comment in my PRs, i'll sink some time into seeing what on earth is going on!